src/allmydata/test/common_util.py

```diff
@@ -1,4 +1,14 @@
+from __future__ import absolute_import
+from __future__ import division
 from __future__ import print_function
+from __future__ import unicode_literals
+
+from future.utils import PY2, bord, bchr, binary_type
+if PY2:
+    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
+
+    # XXX this is a hack that makes some tests pass on Python3, remove in the future
+    from ..scripts import runner
```

Comment:
Might be worth validating this `XXX` at this point. Maybe it can be removed now.

Reply:
I ended up removing this, because the Python 3 test suite passes without it.